addons/save-in-project-page/userscript.js

```diff
@@ -0,0 +1,21 @@
+import downloadBlob from "../../libraries/common/cs/download-blob.js";
+
+export default async function ({ addon, console }) {
+  const vm = addon.tab.traps.vm;
+
+  async function download() {
+    const project = await vm.saveProjectSb3();
+    const title = await addon.tab.waitForElement(".project-title");
+    downloadBlob(`${title.innerHTML}.sb3`, project);
+  }
+
+  const downloadButton = document.createElement("button");
+  downloadButton.innerHTML = "Download project";
+  downloadButton.onclick = download;
+  downloadButton.classList = "button sa-download-button";
+
+  const container = document.querySelector(".project-buttons");
+
+  const seeInsideButton = await addon.tab.waitForElement(".see-inside-button");
+  container.insertBefore(downloadButton, seeInsideButton);
```

Comment:
There's an API that can be used here to ensure the elements always load in the same order.
```suggestion
  addon.tab.appendToSharedSpace({ space: "beforeRemixButton", element: downloadButton, order: 2 });
```

Reply:
Remember own projects don't have a remix button, unless the dedicated "remix own projects" addon is enabled

Reply:
well the no matter what i think it should be next to the see inside project button, that way if there's no remix button, its fine, and if there is a remix button (someone else's project or with the addon) the two purple buttons are next to each other

Reply:
working on that now 👍